src/app/main_modules/context.py

```diff
@@ -0,0 +1,6 @@
+import sys
+import os
+from pathlib import Path
+
+rootDirectory = os.path.realpath(Path(__file__).parents[3])
+sys.path.append(rootDirectory)
```

Comment:
if there is no main file in main_modules, context.py is not required. You only need this in a folder with a main file